File: RecipesApp_/RecipesApp.Core/Responses/GetIngredientsResponse.cs
using RecipesApp.Common.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RecipesApp.Models
{
    public class GetIngredientsResponse
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        public UnitMeasureType Measure { get; set; }
        public DateTime CreatedDate { get; set; }
    }
}

File: client/src/app/core/responses/GetIngredientsResponse.ts
import { UnitMeasure } from "../enums/UnitMeasure";

export class GetIngredientsResponse{
    
    constructor(

        public id:string,
        public name:string, 
        public price:number, 
        public quantity:number, 
        public measure: UnitMeasure
    ){}
}